dist-unminified/recorder.js

```diff
@@ -226,6 +232,16 @@ Recorder.prototype.resume = function() {
   }
 };
 
+Recorder.prototype.setSourceGain = function( gain ){
+  gain = parseFloat( gain );
```

Comment:
What is the benefit of doing parseFloat here?

Reply:
I kept getting the following type error without running `gain` through `parseFloat()`
`Uncaught TypeError: Failed to set the 'value' property on 'AudioParam': The provided float value is non-finite.`